README.md

```diff
@@ -18,6 +18,23 @@ steps:
 - run: icoFoam
 ```
 
+### testing mulitple OpenFOAM versions
+
+```yaml
+strategy:
+  matrix:
+    openfoam-version: [2312, 2406, 2412]  # Add other versions here if needed
```

Comment:
```suggestion
    openfoam-version: [11, 12, 2406, 2412]  # Add other versions here if needed
```

Reply:
Hijacking this example to also showcase support for both openfoam.org and openfoam.com. I don't think cross-distribution packages are that common anymore, but I think it's better to show that it _can[ work with.